ProjectMacros.cmake

```diff
@@ -326,20 +326,8 @@ macro(MAKE_SWIG_TARGET NAME SIMPLENAME KEY_I_FILE I_FILES PARENT_TARGET PARENT_S
     endif()
 
 
-    # Add the -py3 flag if the version used is Python 3
-    set(SWIG_PYTHON_3_FLAGS "")
-    if (Python_VERSION_MAJOR)
-      if (Python_VERSION_MAJOR EQUAL 3)
-        set(SWIG_PYTHON_3_FLAGS "-py3;-relativeimport")
-        message(STATUS "${MODULE_NAME} - Building SWIG Bindings for Python 3")
-      else()
-        message(STATUS "${MODULE_NAME} - Building SWIG Bindings for Python 2")
-      endif()
-    else()
-      # Python2 has been EOL since January 1, 2020
-      set(SWIG_PYTHON_3_FLAGS "-py3;-relativeimport")
-      message(STATUS "${MODULE_NAME} - Couldn't determine version of Python - Building SWIG Bindings for Python 3")
-    endif()
+    # The -py3 flag is now deprecated as Python 3 is the default the version used is Python 3
+    set(SWIG_PYTHON_3_FLAGS "-relativeimport")
```

Comment:
Unrelated cleanup: Avoid having multiple warnings from swig saying "-py3 flag is deprecated"